tests/JMS/Serializer/Tests/Serializer/xml/array_empty.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<result>
+  <entry/>
```

Comment:
If I remember well, I generated this files with your serializer.

I'll manage to skip it.